scylla-macros/src/parser.rs

```diff
@@ -0,0 +1,21 @@
+use proc_macro::TokenStream;
+use syn::{DeriveInput, parse2, Data, Fields, FieldsNamed};
+use proc_macro2::Ident;
+
+/// Parsers the tokens_input to a DeriveInput and returns the struct name from which it derives and
+/// the named fields
+pub(crate) fn parse(tokens_input: TokenStream, current_derive: &str) -> (Ident, FieldsNamed) {
```

Comment:
This function would benefit from a more descriptive name. How about `parse_struct_with_named_fields`?